cmake/openssl.cmake

```diff
@@ -2,36 +2,38 @@
 include(ExternalProject)
 
 if(${ARCH} STREQUAL "x86_64")
-  set(OPENSSL_HASH 3ef6bc4e8be049725ca3887f0d235031)
-
-  ExternalProject_Add(openssl_bundle
-          PREFIX openssl
-          URL https://github.com/fwsGonzo/OpenSSL_bundle/releases/download/v1.2/openssl_bundle.tar.gz
-          URL_HASH MD5=${OPENSSL_HASH}
+  ExternalProject_Add(s2n_bundle
```

Comment:
shouldn't the openssl_bundle still be present or is it part of s2n bundle ?

Reply:
S2N requires openssl 1.1, and I got tired of creating 2 bundles, so I made it into one. My s2n_bundle repo has all the deets.